exchangelib/items.py

```diff
@@ -169,8 +169,8 @@ class Item(RegisterMixIn):
         BooleanField('reminder_is_set', field_uri='item:ReminderIsSet', is_required=True, default=False),
         IntegerField('reminder_minutes_before_start', field_uri='item:ReminderMinutesBeforeStart',
                      is_required_after_save=True, min=0, default=0),
-        CharField('display_cc', field_uri='item:DisplayCc', is_read_only=True),
-        CharField('display_to', field_uri='item:DisplayTo', is_read_only=True),
+        TextField('display_cc', field_uri='item:DisplayCc', is_read_only=True),
+        TextField('display_to', field_uri='item:DisplayTo', is_read_only=True),
```

Comment:
I wonder why these were CharFields to begin with -- maybe because they seemed like they would be short fields? The only question is can you think of a time where we would need the CharField.clean method? I couldnt find any but just wanted to check.

Reply:
I guess they assumed `to` and `cc` fields didnt get too long? but this seems to be popping up a decent amount in the logs, so that assumption is not checking out. And it looks like `clean()` on the `CharField` only does length checking, which is the part we dont want anyway.